src/processors/operation-processor.ts

```diff
@@ -3,13 +3,14 @@ import Resource from "../resource";
 import { Operation, ResourceConstructor } from "../types";
 
 export default class OperationProcessor<ResourceT = Resource> {
-  public app: Application;
```

Comment:
Why did you remove Application?

Reply:
Move to the constructor. Previously we iterating through all processors instances and injecting the app directly on the object like` processor.app = this` in application constructor. Right now we dont have to do it as we have an access to the processor initializer

Reply:
Don't declare properties in the constructor, don't get clever. Use `protected` in the class definition and then set the value in the constructor.

Reply:
Ok, Im new to typescript and thought that this is the way now as I saw that in TS doc